core-web/libs/edit-content/src/lib/fields/dot-edit-content-relationship-field/dot-edit-content-relationship-field.component.html

```diff
@@ -54,7 +56,10 @@
             </td>
             <td style="min-width: 8rem">{{ item.language }}</td>
             <td style="min-width: 8rem">
+                {{ item.modDate }}
+                <!--
                 <p-chip [styleClass]="item.state.styleClass" [label]="item.state.label" />
+                -->
```

Comment:
Will this `chip` be used?

Reply:
Yes, in another card we're going to use the chip component.